### PR TITLE
9 design database

### PR DESCRIPTION
**Pull request by brandhawa99**
got the friend schema thing from stackoverflow 
user model is probably complete 
add stuff to the post model not sure what else we would need

### CODE REVIEW

**Comment by rdhillon1016** [comment]
How are we going to store the images?

File: api/models/Users.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ const UserSchema = new Schema({
   password:{type:String,required:true, minlength:2},
   profilePicUrl:{type:String},
   date_created:{type:Date, required:true,default:Date.now()},
+  dob:{type:Date, required:true},
+  friends:[{type:Schema.Types.ObjectId, ref:'Friends'}]
```
Comment (rdhillon1016) [inline api/models/Users.js]: add 1 more property for "bio"

File: api/models/Posts.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const PostSchema = new Schema({
+  author :{type:Schema.Types.ObjectId, ref:'Users'},
+  date:{type:Date, required:'true', default:Date.now()},
```
Comment (rdhillon1016) [inline api/models/Posts.js]: single-quotes around 'true'

**Comment by rdhillon1016** [comment]
Models should be singular ("Post" vs "Posts")

**Comment by brandhawa99** [comment]
> How are we going to store the images?
Have to still learn how to do that part 
until then we can just have it be a link to an image

**Pull request merged**